pkg/mount/mount_linux_test.go

```diff
@@ -31,9 +31,9 @@ import (
 //   /dev/sdb is ./testdata/12Kzeros
 //	/dev/sdb1 exists, but is not formatted.
 //
-//   /dev/sdc is ./testdata/gptdisk
-//      /dev/sdc1 exists (EFI system partition), but is not formatted
-//      /dev/sdc2 exists (Linux), but is not formatted
+//   /dev/sdc and /dev/nvme0n1 are ./testdata/gptdisk
```

Comment:
I don't see any setup in mount_integration_test.go that would make this true... can you add that change?

Reply:
Oh, this was *already* true. Wow.